Projects/App/Sources/UI/Organization/OrganizationList/OrganizationListViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  OrganizationListViewController.swift
+//  App
+//
+//  Created by 김태호 on 2022/10/16.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import UIKit
+import Combine
+import SnapKit
+import DesignSystem
+
+final class OrganizationListViewController: UIViewController {
+    
+    // MARK: Properties
+    
+    private var viewModel = OrganizationListViewModel()
+    
+    private var subscriptionSet = Set<AnyCancellable>()
+    
+    private var orgNameArray: [String] = []
+    
+    private let titleLabel = UILabel()
+    private let searchingTextFieldView = ShadowTextFieldView()
+    private lazy var searchingTextField = searchingTextFieldView.textField
+    private let tableView = UITableView()
+    
+    // MARK: LifeCycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+        createLayout()
+        bindUI()
+        searchingTextField.delegate = self
+    }
+    
+    // MARK: Function
+    
+}
+
+extension OrganizationListViewController {
+    
+    // MARK: UI Function
+    
+    private func configureUI() {
+        view.backgroundColor = .white
+        
+        titleLabel.text = "참여할 대학교를 알려주세요"
+        // TODO: system font size로 바꾸기
+        titleLabel.font = UIFont.boldSystemFont(ofSize: 26)
+        
+        tableView.dataSource = self
+        tableView.register(OrganizationListCell.self, forCellReuseIdentifier: OrganizationListCell.identifier)
+    }
+    
+    private func createLayout() {
+        view.addSubviews([titleLabel, searchingTextFieldView, tableView])
+        
+        titleLabel.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(16)
+            make.left.equalTo(view.snp.left).offset(20)
+        }
+        
+        searchingTextFieldView.snp.makeConstraints { make in
+            make.top.equalTo(titleLabel.snp.bottom).offset(16)
+            make.left.equalTo(view.snp.left).offset(20)
+            make.right.equalTo(view.snp.right).offset(-20)
+            make.height.equalTo(45)
+        }
+        
+        tableView.snp.makeConstraints { make in
+            make.top.equalTo(searchingTextFieldView.snp.bottom).offset(16)
+            make.left.equalToSuperview()
+            make.right.equalToSuperview()
+            make.bottom.equalTo(view.safeAreaLayoutGuide.snp.bottom)
+        }
+    }
+    
+    private func bindUI() {
+        self.viewModel.$searchingArray.sink { orgNameArray in
```

Comment:
p1
weak self 필요합니다!
```suggestion
        self.viewModel.$searchingArray
        .sink { [weak self ] orgNameArray in
        guard let self = self else { return }
```

Reply:
++ viewModel의 searchingArray -> view의 orgNameArray 로 데이터를 가져오면 안될 것 같아요

Reply:
130dc01 반영했습니다! 현재는 빌드되어 돌아가도록 해놨는데 구현 부분에 있어서는 좀 더 생각을 해보고 리팩토링 하겠습니다!

Reply:
555462d 변경하였습니다! 모든 로직이 ViewModel에서 돌아가도록 하였습니다